tests/test_result.py

```diff
@@ -34,6 +37,7 @@ def _(*args):
         Field('slotted', String, _),
         Field('tatler', Optional[Record[{'orudis': String}]], _),
         Field('coom', Record[{'yappers': String}], _),
+        Field('lovecraft', LovecraftRecord, _),
```

Comment:
I think we should test also more complex field types here, with `Optional` and `Sequence` types involved.

Reply:
Added more tests in 666e112